api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -108,9 +131,17 @@ message CommonTlsContext {
     CertificateValidationContext default_validation_context = 1
         [(validate.rules).message = {required: true}];
 
-    // Config for fetching validation context via SDS API.
-    SdsSecretConfig validation_context_sds_secret_config = 2
-        [(validate.rules).message = {required: true}];
+    // Config for fetching validation context via SDS API. Note SDS API allows certificates to be
+    // fetched/refreshed over the network "asynchronously" with respect to the TLS handshake.
```

Comment:
Nit: why quotes around "asynchronously"? This seems to imply it's not really async?

Reply:
removed quotes